frontend/src/scenes/data-management/definition/DefinitionView.tsx

```diff
@@ -32,6 +35,34 @@ export const scene: SceneExport = {
     }),
 }
 
+type StatusProps = {
+    tagType: LemonTagType
+    label: string
+    icon: React.ReactNode
+    tooltip: string
+}
+
+const statusProps: Record<PropertyDefinitionVerificationStatus, StatusProps> = {
+    verified: {
+        tagType: 'success',
+        label: 'Verified',
+        icon: <IconCheckCircle />,
+        tooltip: 'This property is verified and can be used in filters and other selection components.',
+    },
```

Comment:
Probably out of scope, but would be a big value add to actually display the "verified" checkmark in the list itself, not only the popover: 
<img width="908" alt="Screenshot 2025-02-25 at 22 56 30" src="https://github.com/user-attachments/assets/ba6b2590-1d67-4dc1-ab72-35c07ca13f43" />

Reply:
Yeah good idea. I changed the UX here a bit and did a secondary icon thing to keep it tidy:

<img width="905" alt="Arc 2025-02-25 20 28 26" src="https://github.com/user-attachments/assets/7822490c-b2bb-4e00-b70c-22a21ef496f3" />

It's also in use on the property list page and IMO is cleaner than having the double icons bracketing the prop / event name:

| Before | After |
|--------|--------|
| <img width="437" alt="image" src="https://github.com/user-attachments/assets/c7ff79b6-4ce6-40b4-a5b5-7ecba8f9cd4a" /> | <img width="354" alt="Arc 2025-02-25 20 27 47" src="https://github.com/user-attachments/assets/e27887a9-7a2f-420d-8f0c-ad492a1fe50c" /> |